Makefile

```diff
@@ -13,9 +13,7 @@ GOFMT_CHECK=$(shell find . -not \( \( -wholename './.*' -o -wholename '*/vendor/
 REV=$(shell git describe --long --tags --match='v*' --always --dirty)
 
 # Upstream tuned daemon variables
-TUNED_REPO:=https://github.com/redhat-performance/tuned.git
-TUNED_COMMIT:=954bc4624db8fb345e9fb264ee2ac09d1736105a
-TUNED_DIR:=daemon
+TUNED_COMMIT:=HEAD
```

Comment:
For the upstream NTO build, don't we want to pick a specific tag in the tuned repo to build from?

Reply:
@jmencak

Reply:
Yes, we do. That's why we keep the `TUNED_COMMIT` variable and it is meant to be used.  It is a lot of maintenance to keep the variable up-to-date in the Makefile.  That's why I've decided to set this to HEAD.  Other thought welcome, but we haven't keep up-to-date with this in the past, which resulted in old `TuneD` in the repo.  I believe that keeping this to HEAD is lesser of the two evils, but I can be wrong.

Reply:
OK - so you are saying that HEAD will be set by something else when the upstream NTO build is done?

Reply:
Thanks Jiri - I think it makes sense to use HEAD when building NTO from the master branch. The one tricky part (in the future) might be when someone wants to build upstream NTO from a specific branch (e.g. release-4.xx). In that case, it may be hard to figure out what tuned commit/tag is appropriate and using HEAD will probably not be the right choice. To make it easier, it might be nice to set the TUNED_COMMIT here to an appropriate tag when a new NTO branch is created, but we probably won't remember.